R/remove.missing.levels.lm.R

```diff
@@ -0,0 +1,77 @@
+#' @title remove.missing.levels.lm
```

Comment:
Because so far it is only applied to lm* models. I changed the name to `missingLevelsTrain()`. Better?